Tests/FoundationTests/NSAttributedStringExtensionsTests.swift

```diff
@@ -306,6 +306,24 @@ final class NSAttributedStringExtensionsTests: XCTestCase {
         #endif
     }
 
+    // MARK: - Initializer
+    
+    func testInitImage() {
+        #if canImport(UIKit)
+        let font = UIFont.systemFont(ofSize: 19)
+        let attrString = NSMutableAttributedString(string: "Hello Bitcoin!", attributes: [.font: font])
+        let sizeBefore = attrString.size()
+        let bundle = Bundle(for: NSAttributedStringExtensionsTests.self)
+        let image = UIImage(named: "TestImageInitAttributedString", in: bundle, compatibleWith: nil)!
+        let imageString = NSAttributedString(image: image, font: font)
+        attrString.insert(imageString, at: 6)
+        let sizeAfter = attrString.size()
+        
+        let calculatedSize = CGSize(width: sizeBefore.width + image.size.width, height: sizeBefore.height)
+        XCTAssertEqual(calculatedSize, sizeAfter)
```

Comment:
I'm not sure that I understand the test. You've added an image, but you're testing that the height is still that of the original text? What if the image has a larger height than the text? Will the size still match the text, or will the image height become the new size height?

Reply:
I've added `max(sizeBefore.height, image.size.height)`